mutmut/__main__.py

```diff
@@ -257,10 +261,6 @@ def pragma_no_mutate_lines(source):
 def create_mutants_for_file(filename, output_path):
     input_stat = os.stat(filename)
 
-    if output_path.exists() and output_path.stat().st_mtime == input_stat.st_mtime:
```

Comment:
What is this deletion? This PR should be about just copying the file tree as the title says.

Reply:
I explained it in a previous comment but if that code is there it doesn't create the mutants because `output_path.stat().st_mtime == input_stat.st_mtime` will be true so either this logic needs to be modified or it removed for it to work with the `shutil.copytree`

Reply:
Ah. My bad.